src/main/java/com/salesforce/dev/framework/DataDrivenManager.java

```diff
@@ -109,4 +110,33 @@ public DataDrivenManager(){
         }
         return campaignsArray.iterator();
     }
+
+    /*Returns chatter properties
+    * @param fileJson
+    * @return Iterator<Chatter[]>
+    * */
+    public Iterator<Chatter[]> getChatter(String fileJson) {
+
+        Collection<Chatter[]> chattersArray = new ArrayList<Chatter[]>();
+        try {
+            parser = new JSONParser();
+            Object jsonObject = parser.parse(new FileReader("src/test/resources/" + fileJson));
+            JSONArray jsonArray = (JSONArray) jsonObject;
+            ObjectMapper objectMapper = new ObjectMapper();
+            List<Chatter> navigation = objectMapper.readValue(jsonArray.toJSONString(),
+                    objectMapper.getTypeFactory().constructCollectionType(
+                            List.class, Chatter.class));
+
+            for (Chatter chatter : navigation)
+                chattersArray.add(new Chatter[]{chatter});
+
+        } catch (FileNotFoundException e) {
+                LoggerManager.getInstance().addErrorLog(this.getClass().getName(), "File not found for chatter - Json file:", e);;
+        } catch (IOException e) {
+            LoggerManager.getInstance().addErrorLog(this.getClass().getName(), "Error on data for Chatter from Json file:", e);
+        } catch (ParseException e) {
+            LoggerManager.getInstance().addErrorLog(this.getClass().getName(), "Error on data for Chatter:", e);
```

Comment:
Notice that the logger is duplicated several times
```java
 LoggerManager.getInstance().addErrorLog(this.getClass().getName(), 
```

Could you refactor that?
something like this

```java
\\class attribute 
private static final LoggerManager LOGGER = LoggerManager.getInstance().setClassName(this.getClass().getName());


try {
\\.....
} catch(FileNotFoundException e) {
     LOGGER.addErrorLog("File not found for chatter - Json file:", e);
} 
\\.......
```